src/TokenizedStrategy.sol

```diff
@@ -812,8 +884,9 @@ contract TokenizedStrategy {
         uint256 maxLoss
     ) private returns (uint256) {
         require(receiver != address(0), "ZERO ADDRESS");
-        require(shares <= maxRedeem(owner), "ERC4626: withdraw more than max");
+        require(maxLoss <= MAX_BPS, "max loss");
```

Comment:
i'm not sure this is a helpful error message

Reply:
https://github.com/yearn/tokenized-strategy/pull/64/commits/b2502500505dd6c90f651b4ff27db10b4033f1ec